packages/core/src/components/button/button-mixin.scss

```
@@ -103,7 +103,7 @@
   width: 100%;
   height: 100%;
 
-  padding: 0 0.5rem;
+  padding: var(--ix-button-padding, 0 0.5rem);
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)